x/pricefeed/client/rest/queryMarket.go

```diff
@@ -1,23 +0,0 @@
-package rest
```

Comment:
@jununifi 
What happens if I delete the rest file?
If you have documentation, could you post the URL or something?